src/status_im2/contexts/onboarding/enter_seed_phrase/view.cljs

```diff
@@ -0,0 +1,75 @@
+(ns status-im2.contexts.onboarding.enter-seed-phrase.view
+  (:require [quo2.core :as quo]
+            [quo.core :as quo1]
+            [quo2.foundations.colors :as colors]
+            [clojure.string :as string]
+            [status-im.ethereum.mnemonic :as mnemonic]
```

Comment:
ideally we can migrate this file to status_im2.
I guess it goes in status_im2.common.ethereum.mnemonic